client/src/templates/Challenges/video.css

```diff
@@ -72,11 +72,6 @@
   border-bottom: 4px solid var(--tertiary-background);
 }
 
-.video-quiz-input-hidden {
-  position: absolute;
-  left: -9999px;
-}
-
```

Comment:
We can't get rid of this yet because the MC questions in C# and Python courses use this class. But my PR #51748 removes this class from those questions.

Reply:
Your PR has gone through. 😁